docs/how-to/ccv0.sh

```diff
@@ -187,6 +189,21 @@ configure() {
     debug_function enable_full_debug
     # Temp PoC verify code: Inject policy path config parameter
     sudo sed -i -e 's%^kernel_params = "\(.*\)"%kernel_params = "\1 agent.container_policy_file=/etc/containers/quay_verification/quay_policy.json"%g' /etc/kata-containers/configuration.toml
+
+    # K8s doesn't fully work with kata cc in this enviornment yet issues #3511
+    if [ "${KUBERNETES}" != "yes" ]; then
+        # insert the cri_handler = "cc" into the [plugins.cri.containerd.runtimes.kata] section
+        sudo sed -z -i 's/\([[:blank:]]*\)\(runtime_type = "io.containerd.kata.v2"\)/\1\2\n\1cri_handler = "cc"/' /etc/containerd/config.toml
+    fi
+    
+    # Add cni directory to containerd config
+    echo "    [plugins.cri.cni]
```

Comment:
Doesn't it need admin permission to write to `/etc/containerd/config.toml`?

Reply:
Yes - funny you should mention it. As I comment at the top (at the point I wrote this PR) I'd only ever run it as root for testing (sorry). I have a branch https://github.com/stevenhorsman/kata-containers/tree/ccv0-improvements that I'm going to create a PR for once this is merged that enables it to run as a non-root (but sudo) user, so you could take a look at that instead/as well?

Reply:
Cool. I will have a look and test  #3564